### PR TITLE
Bug 1893637: Fix bug in reflector not recovering from "Too large resource version"

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

File: hack/update-codegen.sh [hunk 1]
```diff
@@ -36,7 +36,7 @@ git clone git@github.com:kubernetes/code-generator.git
 popd
 pushd ${WORK_DIR}/go/src/k8s.io/code-generator
 # The version needs to match the one in Gopkg.toml
-git checkout kubernetes-1.15.0-alpha.0
+git checkout kubernetes-1.18.6
```
Comment (p0lyn0mial) [inline hack/update-codegen.sh:39]: I had to manually `cp -r github.com/kubernetes-sigs/kube-storage-version-migrator/pkg/clients pkg`

**Comment by p0lyn0mial** [comment]
/hold

how do I test it ?

/e2e-aws-operator-encryption

File: go.mod [hunk 2]
```diff
@@ -3,33 +3,31 @@ module github.com/kubernetes-sigs/kube-storage-version-migrator
 go 1.12
```
Comment (sanchezl) [inline go.mod:3]:
```suggestion
go 1.13
```

**Review by sanchezl — COMMENTED** [review]
I'm getting a diff in `pkg/apis/migration/v1alpha1/zz_generated.deepcopy.go` when I run update-codegen.sh

**Comment by sanchezl** [comment]
> how do I test it ?

You'll have to make proof PRs for library-go, kas-o, oas-o.

File: go.mod (hunk 2 above)
Comment (p0lyn0mial) [reply go.mod]: done

**Comment by p0lyn0mial** [comment]
> /test e2e-aws-operator-encryption

for ^ we need https://github.com/openshift/release/pull/13177

**Comment by p0lyn0mial** [comment]
the tests are green, @sanchezl ptal

**Comment by p0lyn0mial** [comment]
/retitle Bug 1893637: Fix bug in reflector not recovering from "Too large resource version"

**Comment by wangke19** [comment]
Launched one 4.5 cluster with this PR successfully, verification steps see https://bugzilla.redhat.com/show_bug.cgi?id=1893637#c1

**Pull request merged**